test/Streamly/Test/Data/Unbox.hs

```diff
@@ -46,6 +65,83 @@ test v1 v2 = monadicIO $ do
     assert (x2 == v2)
     assert (x1 == v1)
 
+testInt :: IntWrap -> Property
+testInt val =
+    monadicIO $ do
+        var <- run $ newUnpinnedBytes (sizeOf (Proxy :: Proxy IntWrap))
+        run $ pokeWith var 0 val
```

Comment:
Do not use peekWith/pokeWith - these are to be removed, there is an issue about it.